core/src/transaction_status_service.rs

```diff
@@ -0,0 +1,78 @@
+use crate::result::{Error, Result};
+use solana_client::rpc_request::RpcTransactionStatus;
+use solana_ledger::{blocktree::Blocktree, blocktree_processor::TransactionStatusBatch};
+use solana_runtime::bank::Bank;
+use std::{
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        mpsc::{Receiver, RecvTimeoutError},
+        Arc,
+    },
+    thread::{self, Builder, JoinHandle},
+    time::Duration,
+};
+
+pub struct TransactionStatusService {
+    t_transaction_status: JoinHandle<()>,
```

Comment:
what's the `t_` prefix?   `trumpet`?

Reply:
Lol, `thread`, I assume. I was following precedent here (repair, replay, retransmit, etc, services), but I see we also have a number of services that don't use this syntax, so I'll just change to be more verbose.